scripts/do-release.sh

```diff
@@ -267,20 +288,9 @@ read_release_file
 extract_semver "${release['version']#v}"
 
 case "${release['status']}" in
-branch)
-    create_branches
-    ;;
-shipyard)
-    release_shipyard
-    ;;
-admiral)
-    release_admiral
-    ;;
-projects)
-    release_projects
-    ;;
-released)
-    release_all
+branch|shipyard|admiral|projects|installers|released)
+    # shellcheck disable=SC2086
+    release_${release['status']}
```

Comment:
This *should* work here:

```suggestion
    "release_${release['status']}"
```